finnish/README.md

```diff
@@ -1,9 +1,26 @@
-# Finish Braille Specifications and Guidelines
+# Finnish Braille Specifications and Guidelines
 
-## [Pistekirjoituksen perusteet](pistek_perusteet.html)
+## [Pistekirjoituksen perusteet (PDF)](Pistekirjoituksen_perusteet.pdf)
 
-- _by_: Finnish Braille Authority and the Finnish Library for the Visually Impaired
+- _description_: 6-dot braille for literary texts
+- _by_: Finnish delegation for braille and Accessibility Library Celia
 - _applies to_: Finland
-- _published_: 2000
-- _language_: Finnish
+- _published_: 2021
+- _language_: Finnish, Finland Swedish
```

Comment:
Is this a different document than it was before? If it's the same, you can put "_published_: 2000, 2021" to indicate that it was originally from 2000, and has been updated in 2021.

Usually we include "_title translated_". In this case it would be:

    _title translated_: Braille basics

That usually voids the need for _description_. If a description is still needed, it's better to put it in a sentence below (inside the section), or possibly in the introduction below "Finnish Braille Specifications and Guidelines".